testem.json

```diff
@@ -1,11 +1,21 @@
 {
   "framework": "mocha",
-  "test_page": "tests/index.html?hidepassed&coverage",
+  "test_page": "tests/index.html?hidepassed",
   "disable_watching": true,
   "launch_in_ci": [
-	"Firefox"
+    "Firefox"
   ],
   "launch_in_dev": [
-	"Chrome"
-  ]
+    "Firefox"
```

Comment:
If I were to run `ember test -server` the tests would run in Chrome and with this change they will run in Firefox. If we have both listed then the tests will run in both. We get use the command to run both and see if there are failures that occur in one or the other as opposed to just one browser type.